x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_eql_search_request.ts

```diff
@@ -87,6 +87,10 @@ export const buildEqlSearchRequest = ({
           filter: requestFilter,
         },
       },
+      // the allow_partial$ query parameters supersede xpack settings on cluster
+      // @ts-expect-error unknown property allow_partial_search_results
+      // TODO: remove this ts-expect when 8.18 elasticsearch client is released.
```

Comment:
Is there a PR/Issue we can reference, here?

Reply:
v8.16.0 package.json: https://github.com/elastic/kibana/blob/d4cc532919d97d6c36e0b12bdf08f5b0cd417cf5/package.json#L118

Reply:
@yctercero @rylnd I just opened this ticket to track the missing type support. Wondering when the 8.18 client will be added to the codebase 🤔

https://github.com/elastic/kibana/issues/208760#issue-2818438716

Reply:
Asking operations: https://elastic.slack.com/archives/C5UDAFZQU/p1738167614946439